RobotArmAPP/Classes/HTTPRequests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace RobotArmAPP.Classes
+{
+    class HTTPRequests
+    {
+        public async Task<string> GetRequest(string eixo1Valor, string eixo2Valor, string eixo3Valor, string eixo4Valor, string garraValor)
+        {
+            try
+            {
+                HttpClient cliente = new HttpClient();
+                string resultado = await cliente.GetStringAsync("http://10.10.10.10:18/?param1=" + eixo1Valor + "&param2=" + eixo2Valor + "&param3=" + eixo3Valor + "&param4=" + eixo4Valor + "&param5=" + garraValor);
```

Comment:
```suggestion
                string resultado = await cliente.GetStringAsync(baseUrl + eixo1Valor + "&param2=" + eixo2Valor + "&param3=" + eixo3Valor + "&param4=" + eixo4Valor + "&param5=" + garraValor);
```
Criar uma propriedade nessa classe que tem a Url, portas, etc. Quando for criar a página de configurações, talvez dar a possibilidade de editar estas configurações.

Reply:
É uma boa!

Reply:
feito!